README

```diff
@@ -0,0 +1,79 @@
+# Demoapp
+
+[![Go Report Card](https://goreportcard.com/badge/github.com/kastenhq/demoapp)](https://goreportcard.com/badge/github.com/kastenhq/demoapp)
+
+## Overview
+This is a Demo app. can be used to demo or testing purposes.
+It's a kubernetes-native picture gallery.
+
+### Design Goals
+
+This application was created as an example for Kubernets-native application
+
+Why this app is kubernetes-native:
+
+1. The application has 2 backend microservices metadata and store.
+2. These services are written in golang.
+3. This application follows API-first pattern.
+4. The application has swagger generated servers and clients.
+
+## Quick Start
+
+### Pre-requisites
+
+- docker
+- golang
+- helm
```

Comment:
Should you also say a cluster or minikube is required?

Reply:
i don't know.... it's probably optional :)